### PR TITLE
Set up basic calculator

### PR DESCRIPTION
**Pull request by chiztechnology**
Hello.
in this pull request, we performed basic operations of the magic calculator.

### Project requirements

To help you concentrate on how to work with React events, we have developed the math logic you need to make the Calculator component work. You can use the files [calculate.js](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/calculate.js) and [operate.js](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js).

- Copy the files calculate.js and operate.js into a logic/ directory in your project. 💯 
- Analyze the files calculate.js and operate.js, be sure you understand what they do (note: you need to add [big.js](https://www.npmjs.com/package/big-js) as dependency). 💯 
- Import the files in your Calculator component. 💯 
- Implement the event handlers you need to use the math logic from calculate.js and operate.js in your React component. 💯 
- Practice what you have learned until now: use state and props, pass props to child components, use lifecycle methods, lift state up. 💯 
- Test your app in the browser. It should be able to perform math operations. 💯

### CODE REVIEW

File: src/components/Display.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
```
Comment (BregornOriginal) [inline src/components/Display.js]: - [ ] Kindly remove this comment

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -1,38 +1,50 @@
+/* eslint-disable */
```
Comment (BregornOriginal) [inline src/components/ButtonPanel.js]: - [ ] Kindly remove this comment

File: src/components/Button.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ import PropTypes from 'prop-types';
 import './Button.css';
```
Comment (BregornOriginal) [inline src/components/Button.js:4]: - [ ] Kindly remove /* eslint-disable react/prop-types */ this comment

File: src/logic/calculate.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
+/* eslint-disable */
```
Comment (BregornOriginal) [inline src/logic/calculate.js]: - [ ] Kindly remove this comment

File: src/logic/operate.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable */
```
Comment (BregornOriginal) [inline src/logic/operate.js]: - [ ] Kindly remove this comment

File: src/App.js [hunk 6]
```diff
@@ -1,10 +1,27 @@
 /* eslint-disable */
```
Comment (BregornOriginal) [inline src/App.js]: - [ ] Kindly remove this comment

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @chiztechnology  

## Good job so far!

## There are some issues that you still need to work on to go to the next project but you are almost there!

cat-driving-serious

## Highlights

✅ Good commit history
✅ Good pull request description

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logic/operate.js (hunk 5 above)
Comment (elmar8287) [reply src/logic/operate.js]: - [ ] Please, consider removing the eslint disabling, as it was suggested by the previous review 👍

File: src/logic/calculate.js (hunk 4 above)
Comment (elmar8287) [reply src/logic/calculate.js]: - [ ] Please, consider removing the eslint disabling, as it was suggested by the previous review 👍

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @chiztechnology.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights

- Good commits history 👍 
 
## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions
-N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by afanimartin — APPROVED** [review]
Hi @chiztechnology, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉 🎉 🎉 
- Changes made as requested 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**